core/store/migrations/migration1599691818/migrate.go

```diff
@@ -0,0 +1,17 @@
+package migration1599691818
+
+import (
+	"github.com/jinzhu/gorm"
+)
+
+// Migrate creates the encrypted_ocr_keys table
+func Migrate(tx *gorm.DB) error {
+	return tx.Exec(`
+		CREATE TABLE encrypted_ocr_private_keys (
+			id SERIAL PRIMARY KEY,
+			encrypted_priv_keys jsonb NOT NULL,
```

Comment:
It would be nice to put the public keys on here... but not essential

Reply:
Done